### DISCUSSION TITLE
Button tap handling delayed due to double tap wait

### DISCUSSION THREAD

**Discussion by CasperVerswijvelt** [Clooos/Bubble-Card#288] [Feature requests]
When tapping a button, the effect is delayed by 250 ms to allow double taps to be registered, even when a double tap action is not set up / wanted . This makes the button feel sluggish in my opinion. We could maybe not do these timeouts when a double tap action is not set up (or explicitly disabled if you want to maintain the double tap: toggle default behaviour)?

https://github.com/Clooos/Bubble-Card/blob/b2cec5d188d827df42da14150a7a9c774ac85277/src/bubble-card.js#L156-L158

I can open make a PR for this if you want, let me know what you think

**Comment by AlexDaniel** [comment]
@CasperVerswijvelt you said you wanted to open a PR for it? This is the first thing that jumped at me, and it's rather terrible.

**Reply by AlexDaniel** [reply]
OK, I made a PR myself :) https://github.com/Clooos/Bubble-Card/pull/1392
